tests/integration_test/core_mqtt_system_test.c

```diff
@@ -212,7 +212,7 @@
  * @brief Time interval in seconds at which an MQTT PINGREQ need to be sent to
  * broker.
  */
-#define MQTT_KEEP_ALIVE_INTERVAL_SECONDS      ( 5U )
+#define MQTT_KEEP_ALIVE_INTERVAL_SECONDS      ( 20U )
```

Comment:
Is it the broker that was disconnecting us? If a value < 30 is given to AWS IoT, the server should treat it as if it were 30 seconds. Does this value need to be increased more?

Reply:
Thanks for sending the [documentation pointing out this behavior in AWS IoT](https://docs.amazonaws.cn/en_us/general/latest/gr/iot-core.html#iot-protocol-limits). I have seen that 20 seconds fixes the issue already, while it was consistently failing for 5. I have tested with 20 and 60 and both were successful. I will make it 60 to make the test more robust.